doc/changes/latest.inc

```diff
@@ -224,6 +224,8 @@ Bug
 
 - Fix placement of extrapolation points in :meth:`mne.Evoked.plot_topomap` and related functions when exactly three channels were used by `Mikołaj Magnuski`_.
 
+- Fix bug in reading annotations in :func:`read_annotations`, which would not accept ";" character.
```

Comment:
by who? please add name. thx

Reply:
Need to add the name here, I'll push a commit